### PR TITLE
EN-6261: new Rest API routes

### PR DESCRIPTION
**Pull request by bogdan-rosianu**
Added 2 more API routes:
_/network/config_ - the configuration of the node (number of shards, consensus group size and so on)
``` JSON
config: {
"erd_chain_id": "ElrondChainID",
"erd_gas_per_data_byte": 1500,
"erd_meta_consensus_group_size": 7,
"erd_min_gas_limit": 50000,
"erd_min_gas_price": 100000000000000,
"erd_num_metachain_nodes": 7,
"erd_num_nodes_in_shard": 7,
"erd_num_shards_without_meta": 3,
"erd_round_duration": 4000,
"erd_shard_consensus_group_size": 5,
"erd_start_time": 1588848691
}
```

_/network/status_ - metrics related to the network status (current epoch, current nonce and so on)
```JSON
status: {
"erd_current_round": 150,
"erd_epoch_number": 1,
"erd_nonce": 150
}
```

### CODE REVIEW

File: api/node/routes.go [hunk 1]
```diff
@@ -76,6 +77,19 @@ func EpochData(c *gin.Context) {
...
+// ConfigData returns data about current configuration
+func ConfigData(c *gin.Context) {
+	ef, ok := c.MustGet("elrondFacade").(FacadeHandler)
+	if !ok {
+		c.JSON(http.StatusInternalServerError, gin.H{"error": errors.ErrInvalidAppContext.Error()})
+		return
+	}
+
+	configMetrics := ef.StatusMetrics().ConfigMetrics()
+
```
Comment (sasurobert) [inline api/node/routes.go:89]: delete empty line

Comment (bogdan-rosianu) [reply api/node/routes.go]: done

**Comment by LucianMincu** [comment]
`/node/config` should be renamed named as `/network/config`, as the values returned are network related (general values for interacting with the protocol) and not with the local node.
`/network` should be better named as `/network/status` 

on proxy `/network/status/shardID` && `/network/config`

**Comment by LucianMincu** [comment]
`erd_gas_per_data_byte: 0,` on system testing.

**Pull request merged**